rapids_pytest_benchmark/rapids_pytest_benchmark/plugin.py

```diff
@@ -430,9 +430,6 @@ def _getHierBenchNameFromFullname(benchFullname):
 
 
 def pytest_sessionfinish(session, exitstatus):
-    if exitstatus != 0:
-        return
```

Comment:
Removing this might be confusing to users that aren't used to seeing a report even when something fails.  I'd recommend adding an option like `--benchmark-report-on-failure`, which should be easy, then just do this instead:
```
if (exitstatus != 0) and not(config.getoption("benchmark_report_on_failure")):
   return
```